app/controllers/screenings_controller.rb

```diff
@@ -0,0 +1,58 @@
+class ScreeningsController < ApplicationController
+  before_action :set_screening, only: %i[show edit update destroy]
+  before_action :set_hall, :set_movie, only: %i[new create edit update]
```

Comment:
you don't need to set those for create and update actions sience you are not using them there for anything